assets/js/lib/analytics/pageview.js

```diff
@@ -1,17 +1,20 @@
 import { useEffect } from 'react';
 import { useLocation } from 'react-router-dom';
 import { capture } from '@lib/analytics';
+import { getFromConfig } from '@lib/config';
 import { useSelector } from 'react-redux';
 import { getUserProfile } from '@state/selectors/user';
 
 export default function PostHogPageView() {
   const location = useLocation();
   const { analytics_enabled } = useSelector(getUserProfile);
+  const webversion = getFromConfig('webversion');
 
   // Track pageviews
   useEffect(() => {
     capture(analytics_enabled, '$pageview', {
       $current_url: window.location.href,
+      webversion,
```

Comment:
I personally prefer `version` or `appVersion` more than `webversion`.
What do you think?
pd: if you change, don't forget changing the `data-ph-...` field as well

Reply:
Lowercase is important: this means `webversion: webversion` in what we send.
Javascript recommends shorthanding (removing right side) if the value is the same.